src/winetricks

```diff
@@ -6682,7 +6685,9 @@ load_dotnet45()
     w_call remove_mono
 
     # See https://appdb.winehq.org/objectManager.php?sClass=version&iId=25478 for Focht's recipe
-    w_call dotnet35
+
+    # Seems unneeded in wine-2.0
```

Comment:
As far as I know, this is only fixed in staging, see https://bugs.winehq.org/show_bug.cgi?id=26757.